Fabric.Identity.API/Quickstart/Account/AccountController.cs

```diff
@@ -53,6 +53,7 @@ public class AccountController : Controller
             DocumentDbUserStore documentDbUserStore,
             ILogger logger,
             IExternalIdentityProviderServiceResolver externalIdentityProviderServiceResolver,
+            AccountService account,
```

Comment:
Minor - can we name this `accountService` for clarity?

Reply:
good point, i'll make that change.